processor/framework.py

```diff
@@ -285,31 +296,55 @@ class HTCondorWorkflow(Task, law.htcondor.HTCondorWorkflow):
     htcondor_request_memory = luigi.Parameter(
         description="Amount of memory(MB) to be requested in HTCondor job submission."
     )
+    htcondor_request_disk = luigi.Parameter(
+        description="Amount of scratch-space(kB) to be requested in HTCondor job submission."
+    )
     htcondor_universe = luigi.Parameter(
-        description="Universe to be set in HTCondor job submission."
+        description="Universe to be set in HTCondor job submission.",
+        significant=False,
     )
     htcondor_docker_image = luigi.Parameter(
         description="Docker image to be used in HTCondor job submission.",
         default="Automatic",
     )
-    htcondor_request_disk = luigi.Parameter(
-        description="Amount of scratch-space(kB) to be requested in HTCondor job submission."
-    )
     bootstrap_file = luigi.Parameter(
-        description="Bootstrap script to be used in HTCondor job to set up law."
+        description="Bootstrap script to be used in HTCondor job to set up law.",
+        significant=False,
     )
     additional_files = luigi.ListParameter(
         default=[],
         description="Additional files to be included in the job tarball. Will be unpacked in the run directory",
+        significant=False,
     )
     remote_source_script = luigi.Parameter(
         description="Script to source environment in remote jobs. Leave empty if not needed. Defaults to use with docker images",
         default="source /opt/conda/bin/activate env",
+        significant=False,
     )
 
     # Use proxy file located in $X509_USER_PROXY or /tmp/x509up_u$(id) if empty
     htcondor_user_proxy = law.wlcg.get_vomsproxy_file()
 
+    # Do not propagate certain parameters via the ".req()" methode
+    exclude_set = {
+        "ENV_NAME",
+        "htcondor_requirements",
+        "htcondor_remote_job",
+        "htcondor_walltime",
+        "htcondor_request_cpus",
+        "htcondor_request_gpus" "htcondor_request_memory",
```

Comment:
I guess here should be a `,` between "htcondor_request_gpus" and "htcondor_request_memory"?

Reply:
You're right, fixed in https://github.com/KIT-CMS/KingMaker/pull/56/commits/e82424fd4536cf31a3bf8b1f3b686079242d404f .